poetry.lock

```diff
@@ -1,7 +1,74 @@
 # This file is automatically @generated by Poetry 1.6.1 and should not be changed by hand.
-package = []
+
```

Comment:
It would be cool if you could share your Poetry experiences next week. I guess the README should also describe how you're using Poetry.

Reply:
Good idea. In the future, I will have to figure out a better way of organizing the README content. For example, we probably don't want to put the "Development" section on PyPI.